eclipse-collections-forkjoin/pom.xml

```diff
@@ -63,7 +63,21 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
+            </plugin>
+
+            <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19</version>
+                <configuration>
+                    <includes>
+                        <include>**/*Test.java</include>
+                    </includes>
+                    <argLine>-XX:-OmitStackTraceInFastThrow -Xms1024m -Xmx2048m @{argLine}</argLine>
+                    <runOrder>random</runOrder>
```

Comment:
Okay, I see difficulty that you faced: undeclared property won't be expanded into empty string by default during [late property evaluation in `maven-surefire-plugin`](http://maven.apache.org/surefire/maven-surefire-plugin/faq.html#late-property-evaluation) , so that in modules where `jacoco-maven-plugin` not executed such as `scala-unit-tests` JVM will fail with message
```
Error: Could not find or load main class @{argLine}
```

This is addressed in https://github.com/Godin/eclipse-collections/commit/e31408936224c68cc859f45c0686467e49d29519#diff-600376dffeb79835ede4a0b285078036R135 by declaring empty property.